azero/scripts/deploy.ts

```diff
@@ -190,13 +195,72 @@ async function main(): Promise<void> {
   }
 
   console.log("Transferring ownership of most to governance...");
-  await most.tx.setOwner(governanceAddress);
-
+  await most.tx.transferOwnership(governanceAddress);
+  console.log("Accepting ownership of most by governance...");
+  //create proposal with transaction acceptOwnership()
+  console.log("Creating proposal...");
+  await governance
+    .withSigner(governance_members[0])
+    .tx.submitProposal(
+      mostAddress,
+      most.contractAbi
+        .findMessage("Ownable2Step::accept_ownership")
+        .selector.toU8a() as any,
+      [],
+      true,
+    );
+  //Members signing proposal...
+  console.log("Members signing proposal...");
+  await governance.withSigner(governance_members[1]).tx.vote(0);
+  await governance.withSigner(governance_members[2]).tx.vote(0);
+  //execute proposal
+  console.log("Executing proposal...");
+  await governance.tx.executeProposal(0);
```

Comment:
Request: extract this as a function, please. It's gonna be a PITA to update otherwise.